src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
The name of the module should describe its purpose. It would be better to use 'allUsers' or 'users' instead.